server/src/test/scala/org/http4s/server/middleware/TimeoutSpec.scala

```diff
@@ -47,19 +47,19 @@ class TimeoutSpec extends Http4sSpec {
     }
 
     "cancel the loser" in {
-      val cancelled = new AtomicBoolean(false)
+      val canceled = new AtomicBoolean(false)
```

Comment:
The travis failure is deserve. Queens english ftw.

Reply:
![](http://i0.kym-cdn.com/photos/images/original/001/143/769/86b.jpg)